sentinel-adapter/sentinel-spring-webflux-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/spring/webflux/SentinelWebFluxFilter.java

```diff
@@ -36,18 +37,18 @@
 
     @Override
     public Mono<Void> filter(ServerWebExchange exchange, WebFilterChain chain) {
-        return chain.filter(exchange)
-            .transform(buildSentinelTransformer(exchange));
-    }
-
-    private SentinelReactorTransformer<Void> buildSentinelTransformer(ServerWebExchange exchange) {
         // Maybe we can get the URL pattern elsewhere via:
         // exchange.getAttributeOrDefault(HandlerMapping.BEST_MATCHING_PATTERN_ATTRIBUTE, path)
-
         String path = exchange.getRequest().getPath().value();
-        String finalPath = Optional.ofNullable(WebFluxCallbackManager.getUrlCleaner())
-            .map(f -> f.apply(exchange, path))
-            .orElse(path);
+
+        String finalPath = WebFluxCallbackManager.getUrlCleaner().apply(exchange, path);
+        if(StringUtil.isEmpty(finalPath)){
```

Comment:
Please reformat your code. e.g. `if (StringUtil.isEmpty(finalPath)) {`

Reply:
Yes, I fixed. Do we have a unified check style file in Ali? We can add it in, so that CI can check out fmt problems, is it better?

Reply:
Good idea. We may improve this later :)